frontend/utils/universalProfile.ts

```diff
@@ -41,6 +37,28 @@ export const getProfileData = async (
     .LSP3Profile as LSP3ProfileMetadata;
 };
 
+export const getProfileBasicInfo = async (
+  contributor: string,
+  rpcUrl: string,
+): Promise<{ upName: string | null; avatar: string | null } | null> => {
```

Comment:
```suggestion
): Promise<{ upName: string | null; avatar: string | null }> => {
```
minor, this no longer returns null object